apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloCodeGenInstallTask.groovy

```diff
@@ -0,0 +1,37 @@
+package com.apollostack.android.gradle
+
+import com.moowork.gradle.node.npm.NpmTask
+import groovy.json.JsonSlurper
+
+class ApolloCodeGenInstallTask extends NpmTask {
+  static final String NAME = "installApolloCodegen"
+  static final String INSTALL_DIR = "node_modules/apollo-codegen"
+  static final String APOLLOCODEGEN_VERSION = "0.9.6"
+
+  public ApolloCodeGenInstallTask() {
+    group = ApolloPlugin.TASK_GROUP
+    description = "Runs npm install for apollo-codegen"
+    File installDir = this.project.file(INSTALL_DIR)
+
+    setArgs(["install", "apollo-codegen@$APOLLOCODEGEN_VERSION"])
+
+    if (!installDir.exists()) {
+      installDir.mkdirs()
+    } else {
+      if (!apolloVersion().equals(APOLLOCODEGEN_VERSION)) {
```

Comment:
The way Gradle decides whether to skip a task or not is by looking at input and output dirs to see if they've changed.

Doing something like `npm install apollocodegen@VERSION --force` will not work if we want to update the npm package in later releases. 
So, this deletes the `node_modules` and reinstalls if `user version != APOLLOCODEGEN_VERSION`